Leanplum-SDK/Classes/LPMessageTemplates.m

```diff
@@ -759,27 +766,20 @@ - (void)closePopupWithAnimation:(BOOL)animated
         finishCallback();
     }
     
-    if ([[context actionName] isEqualToString:LPMT_WEB_INTERSTITIAL_NAME] ||
-        [[context actionName] isEqualToString:LPMT_HTML_NAME] ) {
-        ((UIWebView *)_popupView).delegate = nil;
-        [(UIWebView *)_popupView stopLoading];
-    }
-
-    _popupView = nil;
-    _popupBackground = nil;
-    _acceptButton = nil;
-    _cancelButton = nil;
-    _titleLabel = nil;
-    _messageLabel = nil;
-    _popupGroup = nil;
-    _dismissButton = nil;
-    _overlayView = nil;
-    
     [[NSNotificationCenter defaultCenter] removeObserver:self
                                                     name:UIApplicationDidChangeStatusBarOrientationNotification
                                                   object:nil];
 }
 
+- (void)removeAllViewsFrom:(UIView *)view
+{
+    [view.subviews enumerateObjectsUsingBlock:^(UIView *obj, NSUInteger idx, BOOL *stop) {
```

Comment:
nil check for view/subviews ? Do we need it ?

Reply:
No we don't. In objc if you send a method to nil, it will return nil.
subviews will also not contain nil because NSArray cannot have nil in it.